ds3-autogen-java/src/main/java/com/spectralogic/ds3autogen/java/generators/responseparser/AllocateJobChunkParserGenerator.java

```diff
@@ -35,7 +35,8 @@
     @Override
     public ImmutableList<ResponseCode> toResponseCodeList(
```

Comment:
would it be better to overload this function like this, and than not to change all the place you call it and pass false for pagination?

public ImmutableList<ResponseCode> toResponseCodeList(final ImmutableList<Ds3ResponseCode> ds3ResponseCodes, final String responseName) {
	return toResponseCodeList(ds3ResponseCodes, responseName, false);
}